SportsHub/SportsHub.Application/Authentication/Models/DTOs/ArticleResponseDTO.cs

```diff
@@ -0,0 +1,16 @@
+﻿using SportsHub.Domain.Models;
+
+namespace SportsHub.AppService.Authentication.Models.DTOs
+{
+    public class ArticleResponseDTO
+    {
+        public int Id { get; set; }
+        public string Title { get; set; }
+        public string Content { get; set; }
+
+        public string StateName { get; set; }
+        public string CategoryName { get; set; }
+
+        public virtual ICollection<User> Authors { get; set; }
```

Comment:
I don't think that you need virtual in Response model

Reply:
Fixed